Cesium.Ast/Statements.cs

```diff
@@ -29,7 +29,7 @@ public record AmbiguousBlockItem(string Item1, string Item2) : IBlockItem;
 // 6.8.4 Selection statements
 public record IfElseStatement(Expression Expression, Statement TrueBranch, Statement? FalseBranch) : Statement;
 
-public record SwitchStatement(Expression Expression, Statement Body) : Statement;
+public record SwitchStatement(Expression Expression, CompoundStatement Body) : Statement;
```

Comment:
I honestly sad that I have to deliever bad news for you. https://github.com/ForNeVeR/Cesium/pull/398
And it is in the standard. I was quite surprised when I implementing this.

Reply:
This is the part of #396, not exactly this PR